Resources/views/Collector/db.html.twig

```diff
@@ -257,7 +257,8 @@
         }
 
         function expandAllQueries(button) {
-            var queries = document.getElementsByClassName('query-section'),
+            var conn = button.getAttribute('data-connection'),
+                queries = document.getElementById('queries-' + conn).getElementsByClassName('query-section'),
```

Comment:
`getElementsByClassName()` does not work with IE before 9.0. Not sure what policy is followed for the DoctrineBundle, but probably it's better to choose an alternative solution here.